src/swarm/neo/connection/ConnectionBase.d

```diff
@@ -1034,6 +1048,23 @@ abstract class ConnectionBase: ISelectClient
 
     /***************************************************************************
 
+        Requests to send outgoing data buffered by the OS for the socket
+        immediately. By default outgoing data are buffered to the maximum
+        payload of a TCP frame using the Linux TCP_CORK socket option.
+
+        This method is intended to be called right after `getPayloadForSending`
```

Comment:
I can imagine an alternate approach, where the send delegate specifies (via a return value or out parameter) whether flushing is desired or not.

I'd be interested to see what the usage of both cases looks like.

What do you think?